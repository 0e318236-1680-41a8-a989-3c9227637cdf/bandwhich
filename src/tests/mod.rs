pub mod fakes;
pub mod cases;
