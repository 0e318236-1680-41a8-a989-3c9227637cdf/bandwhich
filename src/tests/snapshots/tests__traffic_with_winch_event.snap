---
source: src/tests/mod.rs
expression: "&terminal_draw_events_mirror[0]"
---
┌Utilization by connection────────────────────────────────────────────────────────────────────┐┌Utilization by process name──────────────────────────────────────────────────────────────────┐
│Connection                                         Processes            Total Bytes Up/Down  ││Process                        Connection Count               Total Bytes                    │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             │└─────────────────────────────────────────────────────────────────────────────────────────────┘
│                                                                                             │┌Utilization by remote ip─────────────────────────────────────────────────────────────────────┐
│                                                                                             ││Remote Address                                     Connection Count     Total Bytes          │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
│                                                                                             ││                                                                                             │
└─────────────────────────────────────────────────────────────────────────────────────────────┘└─────────────────────────────────────────────────────────────────────────────────────────────┘

